backend/chromeApiServer/src/main/java/com/canvas/service/helperServices/CanvasClientService.java

```diff
@@ -34,23 +33,30 @@
 public class CanvasClientService {
 
     // seattleU host: https://seattleu.instructure.com/api/v1
-    public static final String CANVAS_URL = "https://canvas.instructure.com/api/v1";
+    public String canvasUrl;
     public static final String AUTH_HEADER = "Authorization";
 
     private final OkHttpClient okHttpClient;
 
+    @Autowired
+    private Environment env;
+
     // Logger object
     private static final Logger logger = LoggerFactory.getLogger(CanvasClientService.class);
 
 
     /**
      * Constructor with not arguments
      */
-    public CanvasClientService() {
+    @Autowired
+    public CanvasClientService(Environment env) {
```

Comment:
why? we need the Environment variable loaded into this class for it to work. Hence the change. And we are using Autowired annotation so that when the application starts this is the default constructor that is being used. 
For testing we have the constructor on lines 57 onwards. 
will add comments on what the parameters/variables mean.